### ISSUE TITLE
北京 终于又可以用了

### ISSUE THREAD

**Issue by burryxie** [getlantern/lantern#6505]
输入我的邀请码 58RSM5 来获得三个月的蓝灯专业版！

**Comment by hellmaster37** [comment]
hey guys, not working again in Beijing. Just bought the Pro version. Can someone be more professional and fix this?

**Comment by burryxie** [comment]
Yeah not working again on my PC now. I am considering how to take my money back and I don't want to use this shit any more

**Comment by KingFrank** [comment]
not working,if this problem won't  be solved this week, I want my money back!
